CONTRIBUTORS.rst

```diff
@@ -205,6 +205,7 @@ Listed in alphabetical order.
   Tubo Shi                 `@Tubo`_
   Umair Ashraf             `@umrashrf`_                 @fabumair
   Vadim Iskuchekov         `@Egregors`_                 @egregors
+  Vicente G. Reyes         `@reyesvicente`_             @highcenburg
```

Comment:
This is a broken link, you need to add the link destination further down (after line 219):

```
.. _@reyesvicente: https://github.com/reyesvicente
```